CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Brushable Histogram
 
-## 1.1.6 (2018/12/26)
+## 1.2.0 (TBD)
+- Export a css file
+- Ignoring directories that are irrelevant for this component releases.
+  Like unit tests, documentation and storybook files.
+- Update .babelrc file in order to ship the source code without unit
+  tests.
+- Fix Javascript import path.
 - Adds missing check that cause the histogram to blow-up in some cases
 
-## 1.1.5 (2018/12/26) BUMPED
+## 1.1.5 & 1.1.6 (2018/12/26) BUMPED
```

Comment:
1.1.6 was actually removed, no? Why do we have it here?

Reply:
I've reverted that
![image](https://user-images.githubusercontent.com/7489569/51050622-53cc0480-15c9-11e9-8304-cde046381439.png)

Reply:
Yes, we removed it, so it shouldn't be here in the changelog, right?

Reply:
I think it makes more sense to delete the tag and have it consistent between the repository and npm